api/resource/resource_test.go

```diff
@@ -695,322 +695,165 @@ spec:
 	}
 }
 
-func TestSetOriginalNameAndNs(t *testing.T) {
-	input := `apiVersion: apps/v1
-kind: Secret
-metadata:
- name: newName`
-
-	factory := provider.NewDefaultDepProvider().GetResourceFactory()
-	resources, err := factory.SliceFromBytes([]byte(input))
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	res := resources[0]
-	res.SetOriginalName("oldName", false)
-	res.SetOriginalNs("default", false)
-
-	expected := `apiVersion: apps/v1
-kind: Secret
-metadata:
-  annotations:
-    config.kubernetes.io/originalName: oldName
-    config.kubernetes.io/originalNs: default
-  name: newName
-`
-	bytes, err := res.AsYAML()
-	if err != nil {
-		t.Fatal(err)
-	}
-	assert.Equal(t, expected, string(bytes))
-}
-
-func TestGetOriginalName(t *testing.T) {
-	tests := []struct {
+func TestResource_StorePreviousId(t *testing.T) {
```

Comment:
OK, hope we're not losing coverage here :)

Reply:
these were testing methods I've removed, hence the removal of the tests